int32 is_lost
---
int32 is_done
